src/ResourceManager/SubscriptionDefinition/Commands.SubscriptionDefinition/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.Azure.Management.ResourceManager" version="1.6.0-preview" targetFramework="net452" />
```

Comment:
@wilcobmsft once the updated version of the package gets pushed to NuGet, you can update your PR and the CI should get passed the part it is failing at

Reply:
Yep, will do.